test/client/spec/victory-axis/victory-axis.spec.js

```diff
@@ -15,6 +15,15 @@ import { _approximateTextSizeInternal } from "victory-core/es/victory-util/texts
 
 describe("components/victory-axis", () => {
   describe("default component rendering", () => {
+    it("accepts user props", () => {
```

Comment:
Just curious - is there a reason you added this test in the `client/spec` test suite as well? I was under the impression these were going to get phased out once the jest ones were complete.

Reply:
I added it here as a reminder to include this test when we convert the victory axis tests. If it had been a complex test case I couldn't have bothered, but I just needed some basic confirmation that this was working, and we haven't converted this component yet.